build.gradle

```diff
@@ -31,7 +31,7 @@ dependencies {
     // This dependency is used by the application.
     implementation 'com.google.guava:guava:29.0-jre'
     implementation 'org.hibernate:hibernate-core:5.4.21.Final'
-
+    implementation 'org.flywaydb:flyway-core:6.5.5'
```

Comment:
Documentação recomenda criar uma seção pro flyway aqui:
https://flywaydb.org/documentation/gradle/#build-script-single-database

Reply:
Ele também depende que as classes sejam compiladas antes de rodar as migrações, da pra gente por um ```dependsOn``` aqui também.
https://flywaydb.org/documentation/gradle/#java-migrations-and-callbacks

Reply:
@toledompm essa seção pro flyway pode ser implementada depois ou está fora do padrão este implementation do flyway?

Reply:
https://github.com/NicolasFujisawa/fatec-jpa-eventoapp/issues/2